CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.18.1 (Feb, 12, 2022)
```

Comment:
Version change: from 2.17.0 to 2.18.1?  
Does this need to be 2.17.1?

Reply:
@maheshjfrog I'm stacking this PR behind #313 so 2.18.1 is correct